src/modules/service-catalog/ServiceCatalogList.tsx

```diff
@@ -135,17 +135,17 @@ export function ServiceCatalogList({
               searchInputValue={searchInputValue}
             />
           )}
-          {serviceCatalogItems.length > 0 && (
+          {meta && (meta.before_cursor || meta.after_cursor) && (
```

Comment:
Nit: would `meta?.before_cursor || meta?.after_cursor` be enough?

Reply:
I included the `meta` there to avoid using the `?` both here and when fetching the cursors for the Next and Previous elements. By having it like this, we can infer that `meta` is not null in this scope.
Not sure which version is the better approach or common practice. Let me know what you think @luis-almeida.